src/main/java/uk/gov/crowncommercial/dts/scale/cat/service/csvfileutils/RfxTemplateMappingCSVService.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.crowncommercial.dts.scale.cat.service.csvfileutils;
+
+import java.io.IOException;
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.web.multipart.MultipartFile;
+
+import uk.gov.crowncommercial.dts.scale.cat.model.jaggaer.RfxTemplateMapping;
+import uk.gov.crowncommercial.dts.scale.cat.repo.RfxTemplateMappingRepo;
+import uk.gov.crowncommercial.dts.scale.cat.utils.CSVFileUtils;
+
+@Service
+public class RfxTemplateMappingCSVService {
+
+	  @Autowired
+	  RfxTemplateMappingRepo repository;
+
+	  public void save(MultipartFile file) {
+	    try {
+	      List<RfxTemplateMapping> mappings = CSVFileUtils.csvToRfxTemplateMappings(file.getInputStream());
+	      repository.saveAll(mappings);
+	    } catch (IOException e) {

```

Comment:
I wouldn't catch the IOException - it is perfectly normal for IO operations on service methods like `save()` to throw IOExceptions. I would just let this propagate out and the caller can deal with it.